packages/plugins/rrweb-plugin-console-record/package.json

```
@@ -53,6 +53,7 @@
     "puppeteer": "^20.9.0"
   },
   "peerDependencies": {
-    "rrweb": "^2.0.0-alpha.17"
+    "rrweb": "^2.0.0-alpha.17",
+    "@rrweb/utils": "^2.0.0-alpha.17"
```

Release was just done so we need to reference .18
```suggestion
    "rrweb": "^2.0.0-alpha.18",
    "@rrweb/utils": "^2.0.0-alpha.18"
```